### ISSUE TITLE
Tab Bar Layout

### ISSUE THREAD

**Issue by i028** [minbrowser/min#2314]
The current layout of the top tab is not easy to operate. When moving with the mouse, it is necessary to look for the movable area in the upper left corner. Refer to the top tab of the safari browser.
![image](https://github.com/minbrowser/min/assets/77450618/89ac2fbb-bce2-4ac3-877e-4eea63021162)

**Comment by PalmerAL** [comment]
Yeah, I agree it's suboptimal. How would you propose doing it? We could make the area larger, but I think it looks strange with the current tab design:

<img width="1180" src="https://github.com/minbrowser/min/assets/10314059/a596c0ca-bfb7-496a-bfae-0123e2827d52">

**Comment by sndrkrly** [comment]
True, it's hard to find a place to move the window. I think maybe we can decrease the sizes of the opened tabs? 
I don't have any other idea.

**Comment by PalmerAL** [comment]
We could add some space at the top:

<img width="1173" src="https://github.com/minbrowser/min/assets/10314059/5c48737a-f058-4a94-9602-5a70d8cff178">

I think it looks bad though, because now the tabs don't line up with the traffic lights.

**Comment by sndrkrly** [comment]
> We could add some space at the top:
> 
> 
> 
> <img width="1173" src="https://github.com/minbrowser/min/assets/10314059/5c48737a-f058-4a94-9602-5a70d8cff178">
> 
> 
> 
> I think it looks bad though, because now the tabs don't line up with the traffic lights.

I meant the opened windows tab sizes not the top bar size

**Comment by iambudi** [comment]
Hi @PalmerAL @i028 I agree that it's not optimal and can be painful especially when using a wide screen monitor to frequently move window with a lot of tabs open:
1. Press cmd or ctrl while dragging to move min window. I rarely move individual tabs tho. I can still rearrange them later in the tasks if i want.
_Depending on the effort required to adjust this feature, we can also make dragging any tab as default behaviour to move min window and cmd+drag to move tabs._
2. Add option to disable moving tabs by dragging. This way, i can freely use the entire tabs title area as a dragging area to move min window.

**Comment by PalmerAL** [comment]
> Depending on the effort required to adjust this feature, we can also make dragging any tab as default behaviour to move min window and cmd+drag to move tabs.

This used to be the default behavior actually. Then Electron changed their implementation so that if you have a draggable region, it doesn't receive click events (so clicking on tabs doesn't work): https://github.com/electron/electron/pull/38208#issuecomment-1567405095

> Press cmd or ctrl while dragging to move min window

I think this would be possible though; we would listen for the key down event and activate the draggable region when it happens.
